src/dmd/link.d

```diff
@@ -745,7 +745,7 @@ version (Windows)
         {
             if ((len = strlen(args)) > 255)
             {
-                char* q = cast(char*)alloca(8 + len + 1);
+                const char* q = cast(char*)alloca(8 + len + 1);
                 sprintf(q, "_CMDLINE=%s", args);
```

Comment:
```
dmd\link.d(749): Error: function `core.stdc.stdio.sprintf(scope char* s, scope const(char*) format, ...)` is not callable using argument types `(const(char*), string, const(char)*)`
dmd\link.d(749):        cannot pass argument `q` of type `const(char*)` to parameter `scope char* s`
```

We probably should change the declaration in druntime to reflect this, but I guess for now it looks you can't easily do this here.